specification/automation/resource-manager/Microsoft.Automation/preview/2017-05-15-preview/job.json

```diff
@@ -0,0 +1,613 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AutomationManagement",
+    "version": "2017-05-15-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}/jobs/{jobName}/output": {
+      "get": {
+        "tags": [
+          "Job"
+        ],
+        "operationId": "Job_GetOutput",
+        "description": "Retrieve the job output identified by job name.",
+        "produces": [
+          "text/plain"
+        ],
+        "externalDocs": {
+          "url": "http://aka.ms/azureautomationsdk/joboperations"
+        },
+        "x-ms-examples": {
+          "Get Job Output": {
+            "$ref": "./examples/job/getJobOutput.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../stable/2015-10-31/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../stable/2015-10-31/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/automationAccountName"
+          },
+          {
+            "name": "jobName",
+             "description": "The name of the job to be created.",
+             "type": "string",
+             "required": true,
+             "in": "path"
+          },          
+          {
+            "$ref": "../../stable/2015-10-31/definitions.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/clientRequestId"
+          }
+        ],
+        "responses": {
```

Comment:
what does this return? does it not have a defined schema.

Reply:
It returns the output from the script as generated by Powershell. For example, a script with only statement as { get-date } will generate the following output: Saturday, February 17, 2018 10:09:38 PM

whereas, the script like { "hello"; get-date; "bye"; } will generate the output: 
hello

Saturday, February 17, 2018 10:11:15 PM
bye